### ISSUE TITLE
PDF text is presented correctly but copied text does not match content, leading to garbled text when converting to doc

### ISSUE THREAD

**Issue by justin-hackin** [colinhemphill/nextjs-resume#207]
⚠️ it's a common practice for employers and recruiters to convert PDF to doc. When they do this, the resumes from this project will have text that is mangled (it will also distort text copied from within the PDF).

This is an issue with `react-pdf` and it looks like there's a workaround [here](https://github.com/diegomura/react-pdf/issues/1950#issuecomment-1329042576) that requires modifying the font files. As reported there, I couldn't get them to work with the SourceSansPro so I'd recommend changing the font you're using. I tried a couple that made "Contact information" overflow the sidebar but Lato seems to work nicely.

**Comment by colinhemphill** [comment]
@justin-hackin This one's a bit lower priority for me at the moment, and I'm trying to switch over the default font to Albert to get parity between the web and PDF versions. I did the `fontforge` conversion on the TTF files that get loaded, but I don't think it's working for that font either. It doesn't seem like that's really a valid fix for the issue so I'm not expecting to switch over to something like Lato just yet. Keep me in the loop if you figure out anything.

**Comment by colinhemphill** [comment]
I'm going to consider this a dependency problem and close the issue unless we see some movement on their end. Until then I'll leave it up to individuals to switch to a different font if it's important to them

**Comment by justin-hackin** [comment]
Though there were some improvements when I changed to Lato, the text was still distorted a bit e.g. "W" replaced with "•" 😕. I read more generally this issue [occurs with embedded fonts](https://www.foxit.com/blog/what-to-do-when-weird-characters-appear-in-pdf-files/). I tried removing the custom font altogether and it fixed the issue. Since often the only purpose of a resume file is for robots to read it, I would advise doing the same here.
